frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/data-volume-wrapper.ts

```diff
@@ -148,8 +148,9 @@ export class DataVolumeWrapper extends K8sResourceObjectWithTypePropertyWrapper<
   protected sanitize(type: DataVolumeSourceType, { name, namespace, url }: CombinedTypeData) {
     switch (type) {
       case DataVolumeSourceType.HTTP:
-      case DataVolumeSourceType.REGISTRY:
         return { url };
+      case DataVolumeSourceType.REGISTRY:
+        return { url: `docker://${url}` };
```

Comment:
I wonder if it is a good idea to hardcode this here

Reply:
not sure either, but we can try and see how it goes